scripts/synthetic_59_target_pair_soak.py

```
@@ -0,0 +1,656 @@
+                "-sf",
+                "-X",
+                "POST",
+                f"{base}/api/csv/import/package",
+                "-H",
+                f"Authorization: Bearer {token}",
+                "-F",
+                f"file=@{package_zip}",
+                "-o",
+                tf.name,
+            ]
+            subprocess.run(cmd, check=False)
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)